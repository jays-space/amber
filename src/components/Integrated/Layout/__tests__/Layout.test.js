import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Layout } from "..";

describe("<Layout />", () => {
  it("should render correctly", () => {
    render(<Layout />);

    const layoutComponent = screen.getByTestId("dashboard");
    expect(layoutComponent).toBeInTheDocument();

    expect(layoutComponent).toMatchSnapshot();
  });
});
