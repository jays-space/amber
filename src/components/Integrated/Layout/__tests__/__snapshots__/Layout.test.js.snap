// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render correctly 1`] = `
<div
  class="min-w-full min-h-screen flex flex-row"
  data-testid="dashboard"
>
  <div
    class="md:w-1/4 lg:w-1/6 p-6 bg-slate-900 transform ease-out duration-300 cursor-pointer hidden md:block z-50"
    data-testid="sidebar"
  >
    <div
      class="relative mt-2 mb-12 px-4"
    >
      <img
        alt="amber software logo"
        class="opacity-100"
        data-testid="logo"
        src="/logo.png"
      />
      <div
        class="absolute -top-2 hidden md:flex -right-11 w-10 aspect-square flex items-center justify-center rounded-full bg-yellow-400 hover:bg-yellow-300 active:bg-slate-200 transform ease-out duration-100 cursor-pointer"
        title="toggle-sidebar"
      >
        <svg
          baseProfile="tiny"
          class="text-slate-500 !text-slate-700"
          data-testid="icon-minimize"
          fill="currentColor"
          height="28"
          stroke="currentColor"
          stroke-width="0"
          version="1.2"
          viewBox="0 0 24 24"
          width="28"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6.121 13c-.553 0-1 .448-1 1s.447 1 1 1h1.465l-3.293 3.293c-.391.391-.391 1.023 0 1.414.195.195.451.293.707.293s.512-.098.707-.293l3.414-3.414v1.707c0 .552.447 1 1 1s.879-.448.879-1v-5h-4.879zM7 11c.552 0 1-.448 1-1v-2h2c.553 0 1-.448 1-1s-.447-1-1-1h-3.999l-.001 4c0 .552.447 1 1 1zM17 13c-.553 0-1 .448-1 1v2h-2c-.553 0-1 .448-1 1s.447 1 1 1h4v-4c0-.552-.447-1-1-1zM18.293 4.293l-3.293 3.293v-1.586c0-.552-.447-1-1-1s-1 .448-1 1v5h5c.552 0 1-.448 1-1s-.447-1-1-1h-1.586l3.293-3.292c.391-.391.391-1.023 0-1.414s-1.023-.392-1.414-.001z"
          />
        </svg>
      </div>
    </div>
    <div
      class="mb-8"
    >
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-home"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5 22h14a2 2 0 0 0 2-2v-9a1 1 0 0 0-.29-.71l-8-8a1 1 0 0 0-1.41 0l-8 8A1 1 0 0 0 3 11v9a2 2 0 0 0 2 2zm5-2v-5h4v5zm-5-8.59 7-7 7 7V20h-3v-5a2 2 0 0 0-2-2h-4a2 2 0 0 0-2 2v5H5z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          home
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-inbox"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <polyline
            points="22 12 16 12 14 15 10 15 8 12 2 12"
          />
          <path
            d="M5.45 5.11L2 12v6a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2v-6l-3.45-6.89A2 2 0 0 0 16.76 4H7.24a2 2 0 0 0-1.79 1.11z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          messages
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-calendar"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 1024 1024"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M928 224H768v-56c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v56H548v-56c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v56H328v-56c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v56H96c-17.7 0-32 14.3-32 32v576c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V256c0-17.7-14.3-32-32-32zm-40 568H136V296h120v56c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-56h148v56c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-56h148v56c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-56h120v496zM416 496H232c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h184c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8zm0 136H232c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h184c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8zm308.2-177.4L620.6 598.3l-52.8-73.1c-3-4.2-7.8-6.6-12.9-6.6H500c-6.5 0-10.3 7.4-6.5 12.7l114.1 158.2a15.9 15.9 0 0 0 25.8 0l165-228.7c3.8-5.3 0-12.7-6.5-12.7H737c-5-.1-9.8 2.4-12.8 6.5z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          calendar
        </p>
      </div>
    </div>
    <div
      class="mb-8"
    >
      <p
        class="tracking-wide  text-slate-700 mb-2 px-4 font-semibold uppercase text-sm !tracking-widest text-slate-600 opacity-100"
        data-testid="paragraph"
      >
        development
      </p>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-task"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M139.61 35.5a12 12 0 0 0-17 0L58.93 98.81l-22.7-22.12a12 12 0 0 0-17 0L3.53 92.41a12 12 0 0 0 0 17l47.59 47.4a12.78 12.78 0 0 0 17.61 0l15.59-15.62L156.52 69a12.09 12.09 0 0 0 .09-17zm0 159.19a12 12 0 0 0-17 0l-63.68 63.72-22.7-22.1a12 12 0 0 0-17 0L3.53 252a12 12 0 0 0 0 17L51 316.5a12.77 12.77 0 0 0 17.6 0l15.7-15.69 72.2-72.22a12 12 0 0 0 .09-16.9zM64 368c-26.49 0-48.59 21.5-48.59 48S37.53 464 64 464a48 48 0 0 0 0-96zm432 16H208a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h288a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zm0-320H208a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h288a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16zm0 160H208a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h288a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          tasks
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-project"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 640 512"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M384 320H256c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h128c17.67 0 32-14.33 32-32V352c0-17.67-14.33-32-32-32zM192 32c0-17.67-14.33-32-32-32H32C14.33 0 0 14.33 0 32v128c0 17.67 14.33 32 32 32h95.72l73.16 128.04C211.98 300.98 232.4 288 256 288h.28L192 175.51V128h224V64H192V32zM608 0H480c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h128c17.67 0 32-14.33 32-32V32c0-17.67-14.33-32-32-32z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          projects
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-new"
          fill="currentColor"
          height="24"
          role="img"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title />
          <path
            d="M6.375 23.999c-.95 0-1.95-.749-1.95-2.2v-3.4c0-1.349.85-2.899 2.05-3.548l4.75-2.8-4.75-2.8C5.325 8.5 4.425 7 4.425 5.65V2.2c0-1.45 1-2.2 2.002-2.2.4 0 .849.1 1.249.35l10.7 6.35c.75.45 1.15 1.149 1.15 1.849 0 .75-.452 1.45-1.15 1.85l-2.55 1.5 2.55 1.501c.75.45 1.2 1.15 1.2 1.85 0 .75-.452 1.45-1.2 1.85L7.674 23.65c-.45.25-.85.35-1.3.35zm7.15-10.599l-5.85 3.45c-.45.25-.9 1.05-.9 1.55v3.05l10.15-6zM6.775 2.6v3.05c0 .5.45 1.3.9 1.55l5.85 3.45 3.45-2.05z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          releases
        </p>
      </div>
    </div>
    <div
      class="mb-8"
    >
      <p
        class="tracking-wide  text-slate-700 mb-2 px-4 font-semibold uppercase text-sm !tracking-widest text-slate-600 opacity-100"
        data-testid="paragraph"
      >
        organization
      </p>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-person"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 16 16"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 8a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm2-3a2 2 0 1 1-4 0 2 2 0 0 1 4 0zm4 8c0 1-1 1-1 1H3s-1 0-1-1 1-4 6-4 6 3 6 4zm-1-.004c-.001-.246-.154-.986-.832-1.664C11.516 10.68 10.289 10 8 10c-2.29 0-3.516.68-4.168 1.332-.678.678-.83 1.418-.832 1.664h10z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          employees
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  bg-yellow-500 hover:bg-opacity-80 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 !text-slate-900 mr-4"
          data-testid="icon-people"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 16 16"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15 14s1 0 1-1-1-4-5-4-5 3-5 4 1 1 1 1h8zm-7.978-1A.261.261 0 0 1 7 12.996c.001-.264.167-1.03.76-1.72C8.312 10.629 9.282 10 11 10c1.717 0 2.687.63 3.24 1.276.593.69.758 1.457.76 1.72l-.008.002a.274.274 0 0 1-.014.002H7.022zM11 7a2 2 0 1 0 0-4 2 2 0 0 0 0 4zm3-2a3 3 0 1 1-6 0 3 3 0 0 1 6 0zM6.936 9.28a5.88 5.88 0 0 0-1.23-.247A7.35 7.35 0 0 0 5 9c-4 0-5 3-5 4 0 .667.333 1 1 1h4.216A2.238 2.238 0 0 1 5 13c0-1.01.377-2.042 1.09-2.904.243-.294.526-.569.846-.816zM4.92 10A5.493 5.493 0 0 0 4 13H1c0-.26.164-1.03.76-1.724.545-.636 1.492-1.256 3.16-1.275zM1.5 5.5a3 3 0 1 1 6 0 3 3 0 0 1-6 0zm3-2a2 2 0 1 0 0 4 2 2 0 0 0 0-4z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize !text-slate-900 !font-bold !tracking-wider opacity-100 flex "
          data-testid="paragraph"
        >
          candidates
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-report"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 1024 1024"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M864 518H506V160c0-4.4-3.6-8-8-8h-26a398.46 398.46 0 0 0-282.8 117.1 398.19 398.19 0 0 0-85.7 127.1A397.61 397.61 0 0 0 72 552a398.46 398.46 0 0 0 117.1 282.8c36.7 36.7 79.5 65.6 127.1 85.7A397.61 397.61 0 0 0 472 952a398.46 398.46 0 0 0 282.8-117.1c36.7-36.7 65.6-79.5 85.7-127.1A397.61 397.61 0 0 0 872 552v-26c0-4.4-3.6-8-8-8zM705.7 787.8A331.59 331.59 0 0 1 470.4 884c-88.1-.4-170.9-34.9-233.2-97.2C174.5 724.1 140 640.7 140 552c0-88.7 34.5-172.1 97.2-234.8 54.6-54.6 124.9-87.9 200.8-95.5V586h364.3c-7.7 76.3-41.3 147-96.6 201.8zM952 462.4l-2.6-28.2c-8.5-92.1-49.4-179-115.2-244.6A399.4 399.4 0 0 0 589 74.6L560.7 72c-4.7-.4-8.7 3.2-8.7 7.9V464c0 4.4 3.6 8 8 8l384-1c4.7 0 8.4-4 8-8.6zm-332.2-58.2V147.6a332.24 332.24 0 0 1 166.4 89.8c45.7 45.6 77 103.6 90 166.1l-256.4.7z"
          />
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          report
        </p>
      </div>
      <div
        class="flex  items-center my-2 py-3 px-4 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
      >
        <svg
          class="text-slate-500 text-slate-700 mr-4"
          data-testid="icon-settings"
          fill="currentColor"
          height="24"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M8.686 4l2.607-2.607a1 1 0 0 1 1.414 0L15.314 4H19a1 1 0 0 1 1 1v3.686l2.607 2.607a1 1 0 0 1 0 1.414L20 15.314V19a1 1 0 0 1-1 1h-3.686l-2.607 2.607a1 1 0 0 1-1.414 0L8.686 20H5a1 1 0 0 1-1-1v-3.686l-2.607-2.607a1 1 0 0 1 0-1.414L4 8.686V5a1 1 0 0 1 1-1h3.686zM6 6v3.515L3.515 12 6 14.485V18h3.515L12 20.485 14.485 18H18v-3.515L20.485 12 18 9.515V6h-3.515L12 3.515 9.515 6H6zm6 10a4 4 0 1 1 0-8 4 4 0 0 1 0 8zm0-2a2 2 0 1 0 0-4 2 2 0 0 0 0 4z"
            />
          </g>
        </svg>
        <p
          class="tracking-wide  text-slate-700 capitalize text-white opacity-100 flex "
          data-testid="paragraph"
        >
          settings
        </p>
      </div>
    </div>
  </div>
  <div
    class="w-full bg-slate-100"
    data-testid="main-content"
  >
    <header
      class="relative flex flex-row justify-between items-center px-5 md:px-10 lg:px-20 py-3 bg-white shadow"
      title="header"
    >
      <div
        class="flex flex-1"
      >
        <div
          class="md:hidden"
        >
          <img
            alt="amber software logo"
            class="w-32"
            src="/logo.png"
          />
        </div>
        <div
          class="relative hidden md:flex items-center gap-4"
        >
          <svg
            class="text-slate-500 absolute ml-2 pointer-events-none"
            data-testid="icon-search"
            fill="currentColor"
            height="24"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"
            />
          </svg>
          <input
            class="w-full py-4 pl-10 pr-20 border-none text-xl focus-visible:outline-slate-400"
            placeholder="search..."
            type="search"
          />
        </div>
      </div>
      <div
        class="ml-10 hidden md:flex flex-row items-center"
      >
        <button
          class="w-10 lg:w-14 aspect-square flex justify-center items-center rounded-full last-of-type:mx-4 cursor-pointer transform ease-out duration-100 cursor-pointer bg-yellow-400 hover:bg-opacity-80"
          data-testid="header-button-primary"
        >
          <svg
            class="text-slate-500 !text-slate-800"
            data-testid="icon-add"
            fill="currentColor"
            height="28"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 277.333H277.333V416h-42.666V277.333H96v-42.666h138.667V96h42.666v138.667H416v42.666z"
            />
          </svg>
        </button>
        <button
          class="w-10 lg:w-14 aspect-square flex justify-center items-center rounded-full last-of-type:mx-4 cursor-pointer transform ease-out duration-100 cursor-pointer bg-neutral-200 hover:bg-opacity-80"
          data-testid="header-button-secondary"
        >
          <svg
            class="text-slate-500 "
            data-testid="icon-notification"
            fill="currentColor"
            height="28"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M439.39 362.29c-19.32-20.76-55.47-51.99-55.47-154.29 0-77.7-54.48-139.9-127.94-155.16V32c0-17.67-14.32-32-31.98-32s-31.98 14.33-31.98 32v20.84C118.56 68.1 64.08 130.3 64.08 208c0 102.3-36.15 133.53-55.47 154.29-6 6.45-8.66 14.16-8.61 21.71.11 16.4 12.98 32 32.1 32h383.8c19.12 0 32-15.6 32.1-32 .05-7.55-2.61-15.27-8.61-21.71zM67.53 368c21.22-27.97 44.42-74.33 44.53-159.42 0-.2-.06-.38-.06-.58 0-61.86 50.14-112 112-112s112 50.14 112 112c0 .2-.06.38-.06.58.11 85.1 23.31 131.46 44.53 159.42H67.53zM224 512c35.32 0 63.97-28.65 63.97-64H160.03c0 35.35 28.65 64 63.97 64z"
            />
          </svg>
        </button>
        <img
          alt="duduzile hlongwane's avatar"
          class="w-14 aspect-square rounded-full"
          src="https://media-exp1.licdn.com/dms/image/C4D03AQELG5uq0_LIug/profile-displayphoto-shrink_100_100/0/1637782454380?e=1667433600&v=beta&t=976dwrMZyY3VL1L48bQwAOvR730al8Ad2-vGvz4pgE8"
        />
      </div>
      <div
        class="z-50 md:hidden"
      >
        <button
          class="flex flex-row justify-center items-center mx-2 last-of-type:mr-0 py-2 px-6 text-slate-800 text-base font-bold tracking-widest rounded-md capitalize cursor-pointer hover:bg-opacity-75 transform ease-out duration-100 w-11 aspect-square !p-2 bg-white hover:bg-slate-50 active:bg-slate-200"
          data-testid="button-icon-light"
        >
          <svg
            class="text-slate-500 text-slate-700"
            data-testid="icon-menu"
            fill="currentColor"
            height="24"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
      </div>
      <div
        class="fixed md:hidden top-0 translate-x-full bottom-0 px-10 flex flex-col justify-center items-center w-screen h-screen bg-slate-900 bg-opacity-90 backdrop-blur-md z-40 transition-transform ease-out duration-300"
      >
        <div
          class="mb-0"
        >
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              home
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              messages
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              calendar
            </p>
          </div>
        </div>
        <div
          class="mb-0"
        >
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              tasks
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              projects
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              releases
            </p>
          </div>
        </div>
        <div
          class="mb-0"
        >
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              employees
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  bg-yellow-500 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize !text-slate-900 !font-bold !tracking-wider"
              data-testid="paragraph"
            >
              candidates
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              report
            </p>
          </div>
          <div
            class="flex justify-center items-center my-2 py-3 px-6 rounded-md cursor-pointer  hover:bg-slate-800 transform ease-out duration-100"
          >
            <p
              class="tracking-wide  text-slate-700 capitalize text-white"
              data-testid="paragraph"
            >
              settings
            </p>
          </div>
        </div>
      </div>
    </header>
    <main
      class="h-content p-4 md:p-8 md:px-5 lg:px-10 xl:px-14 2xl:px-20"
    />
  </div>
</div>
`;
